src/common/api/index.ts

```diff
@@ -0,0 +1,12 @@
+import ky from "ky";
+
+export const api = ky.create({
+  prefixUrl: "https://api.example.com",
+  hooks: {
+    afterResponse: [
+      (response) => {
+        response.json();
+      },
+    ],
```

Comment:
오오 이거 의도는 넘 좋은데 파싱한 값 반환이 안 되고 있는 것 같아요.

```suggestion
    afterResponse: [
      async (response) => {
        return await response.json();
      },
    ],
```

이렇게 해야 정상동작할 것 같네요!!!

Reply:
헉 ㅋㅋ 그러네요 꼼꼼한 리뷰 감사합니당 ㅎㅎㅎ

Reply:
이거 알게된 점 공유..ㅎㅎ

1. afterResponse arguments 다시 보니까 요로콤 들어오네욤
(_request, _options, response)

2. return 타입이 아래처럼 고정되어 있어서, 저렇게 사용이 불가했어요 🥲
void | Response | Promise<void | Response>

3. 설령 아래와 같은 작업을 하더라도 어차피 response가 반환된 이후에 받아 실행되는 함수라서, 안에서 새로운 비동기 통신 안할거면 async/await은 사용하지 않아도 되네용
```ts
afterResponse: [
     (_request, _options, response) => {
        console.log(response.json())
      },
    ],
```

Reply:
암튼 json()은 그때그때 fetching 함수에서 사용하는걸루!

Reply:
아하 약간 response 반환 후 후속작업 같은 느낌이군여!!!!! 좋습니당 :)